app.js

```diff
@@ -0,0 +1,39 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const dotenv = require("dotenv");
+dotenv.config(); // 환경변수에 작성해놓았던 TypeORM과 관련된 내용들을 app.js에서 사용할 수 있게 해주는 코드
+
+const { DataSource } = require("typeorm");
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+app.get("/ping", (req, res) => {
+  res.json({ message: "pong" });
+});
```

Comment:
```suggestion
app.get("/ping", (req, res) => {
  return res.status(200).json({ message: "pong" });
});
```
백엔드 api에서 응답에 맞는 적절한 status code까지 핸들링 할 수 있도록 해주세요.